TESTING_AGAINST_AWS.md

```diff
@@ -0,0 +1,98 @@
+# Testing Against AWS - Integration Testing
+
```

Comment:
Problem Statement: This is intended to create controlled and complete integration test fixtures. The 'default' fixture covers .... . The 'minimal' fixture covers ... .